src/lib/components/event/event-summary-table.svelte

```diff
@@ -36,15 +36,15 @@
     <th class="w-44"><EventCategoryFilter {compact} /></th>
     <th class="w-auto xl:w-80">
       <div class="flex w-full justify-end">
-        <button
-          class="relative flex w-28 items-center justify-end rounded sm:justify-between"
+        <Button
```

Comment:
Do we want `Button` styling for any of the filters (e.g. `Date & Time`) in the header too?

![Screenshot 2023-08-29 at 4 42 39 PM](https://github.com/temporalio/ui/assets/15069288/3e37dfd7-ff86-4562-8098-52deeb3d9c40)
![Screenshot 2023-08-29 at 4 42 49 PM](https://github.com/temporalio/ui/assets/15069288/c36c3995-9f0f-40c5-a571-d59e3bdc6ce2)